ldes-consumer/app.py

```
@@ -0,0 +1,591 @@
+        logger.error(f"✗ Error connecting to Docker: {e}")
+        logger.error("Make sure Docker socket is mounted at /var/run/docker.sock")
+        sys.exit(1)
+
+    # Load feeds BEFORE starting containers
+    feeds_config = load_ldes_feeds(config_path)
+
+    # Initialize empty containers list that will be filled after spawning
+    containers = []
+
+    # Setup signal handlers for graceful shutdown
+    def signal_handler(sig, frame):
```

Ruff (0.15.2)

[warning] Unused function argument: `sig`

(ARG001)